src/lib.rs

```diff
@@ -37,8 +37,12 @@ pub(crate) mod naked;
 #[cfg(all(feature = "unwinding", not(target_arch = "arm")))]
 #[allow(unused_extern_crates)]
 extern crate unwinding;
-#[cfg(all(feature = "unwinding", target_arch = "arm"))]
+#[cfg(any(not(feature = "unwinding"), target_arch = "arm"))]
 mod unwind_unimplemented;
+// If we don't have "unwinding", provide stub functions for unwinding and
+// panicking.
+#[cfg(not(feature = "unwinding"))]
+mod stubs;
```

Comment:
I figured given the context it was better to co-locate with the other unwinding related logic?

This was previously tied to the `nightly` feature but has since lost that context. Thus technically `unwind_unimplemented` and `stubs` are similar beyond the `arm` conditional.

Not sure how you want to approach this.